pkg/collect/longhorn.go

```diff
@@ -360,7 +345,7 @@ func GetLonghornReplicaChecksum(clientConfig *rest.Config, replica longhornv1bet
 		Param("stdin", "true").
 		Param("command", "/bin/bash").
 		Param("command", "-c").
-		Param("command", fmt.Sprintf("sha256sum /host/var/lib/longhorn/replicas/%s/*", replica.Spec.DataDirectoryName))
+		Param("command", fmt.Sprintf("if [ -d %s ]; then md5sum %s/*; fi", dir, dir))
```

Comment:
is md5sum actually meaningfully faster than sha256sum?

Reply:
I'd expect to be more limited by disk than CPU

Reply:
I got 1m 21s for shas56sum and 25s for md5sum in my test.